src/meltano/core/bundle/discovery.yml

```diff
@@ -3435,6 +3435,9 @@ transforms:
     pip_url: 'https://gitlab.com/meltano/dbt-tap-google-analytics.git@config-version-2'
     vars:
       schema: "{{ env_var('DBT_SOURCE_SCHEMA') }}"
+    requires:
+      files:
+        - name: dbt
```

Comment:
I don't know if we should be making any changes to `discovery.yml`.

```suggestion
```

My expectation was that we would just add these to Hub. Wdyt?

Reply:
@aaronsteers correct but I'm using `discovery.yml` to mock the Hub, i.e. I'm feeding the mock with this catalog.

Reply:
https://github.com/meltano/meltano/blob/f564cf24c70add9cf12d8b1d5e94486cd117fe72/tests/conftest.py#L72

Reply:
@aaronsteers I'll merge this first thing tomorrow since it's blocking dev/testing of Hub features: https://github.com/meltano/meltano/issues/6046